plugins/woocommerce-admin/client/two-column-tasks/sectioned-task-list.tsx

```diff
@@ -117,7 +122,11 @@ export const SectionedTaskList: React.FC< TaskListProps > = ( {
 		return <div className="woocommerce-task-dashboard__container"></div>;
 	}
 
-	if ( isComplete && keepCompletedTaskList !== 'yes' ) {
+	if (
+		isComplete &&
+		keepCompletedTaskList !== 'yes' &&
```

Comment:
I assume just relying on this causes a delay and we want to optimistically show the task list again before this option is re-retrieved, right?

This is another use case for our generic dismissal API.  Would love to discuss getting this into an upcoming cycle.  cc @mattsherman